bin/plugin-helpers.js

```diff
@@ -34,10 +34,12 @@ program
   .command('build [files...]')
   .description('Build a distributable archive')
   .on('--help', docs('build'))
+  .option('--build-destination <path>', 'Custom target path for the build, relative to the plugin root')
```

Comment:
Nit: Consider providing a short option as well to be consistent with other parameters.